src/renderer/mod.rs

```diff
@@ -349,6 +347,11 @@ impl Renderer {
             DrawCommand::ModeChanged(new_mode) => {
                 self.current_mode = new_mode;
             }
+            DrawCommand::Flush => {
```

Comment:
It looks like a flush is forced after every submitted batch on sending side. Why not just execute a flush at the end of a received batch instead of requiring an extra command?

Reply:
It was simpler this way, since all the batches were combined into one array on the receiving side here https://github.com/neovide/neovide/blob/834c32d9ffe19da613ebcf1c54bc392249cd30b0/src/renderer/mod.rs#L149-L151

But @MultisampledNight has refactored that now, so I will change it.

Reply:
This is now ifxed